README.rst

```diff
@@ -1,61 +1,50 @@
-django-spanner
-==============
+Cloud Spanner support for Django
```

Comment:
Would it be better to have `Cloud Spanner support for Django framework`?

Reply:
I think it's ok as is, django is very popular and I'd expect people looking at this package to know what it is.